prime-router/src/main/kotlin/history/azure/ReportFileFunction.kt

```diff
@@ -191,10 +248,11 @@ abstract class ReportFileFunction(
              */
             fun extractSortCol(query: Map<String, String>): HistoryDatabaseAccess.SortColumn {
                 val col = query["sortcol"]
-                return if (col == null)
-                    HistoryDatabaseAccess.SortColumn.CREATED_AT
-                else
+                // https://stackoverflow.com/a/41844910/1978219
```

Comment:
Remove this link and add a comment to what you want to say

Reply:
done